vendor/github.com/keybase/client/go/auth/credential_authority.go

```diff
@@ -2,11 +2,12 @@ package auth
 
 import (
 	"fmt"
+	"time"
```

Comment:
No `vendor.json` update?  Is this supposed to be here?  Is there a corresponding `client` PR?